frontend/src/pages/Landing/landing.module.css

```diff
@@ -30,6 +30,31 @@
   justify-content: center;
   align-items: center;
   padding: 16px;
+  position: relative;
+}
+
+.dti {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
```

Comment:
i wouldnt absolute position this, thats probably why you're getting the overlapping